test/test_memoryview.py

```diff
@@ -2,11 +2,134 @@
 # coding: utf-8
 
 
+from array import array
 from msgpack import packb, unpackb
+import sys
 
 
-def test_pack_memoryview():
-    data = bytearray(range(256))
-    view = memoryview(data)
-    unpacked = unpackb(packb(view))
-    assert data == unpacked
+# For Python < 3:
+#  - array type only supports old buffer interface
+#  - array.frombytes is not available, must use deprecated array.fromstring
+if sys.version_info[0] < 3:
+    def __memoryview(obj):
```

Comment:
Why do you use double underscore prefix here?
Single underscore is enough (while it's not necessary for test packages since there are no public (stable) API in tests).